webapps/api/helpers.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env python2
+# Copyright (C) 2016 iNuron NV
+#
+# This file is part of Open vStorage Open Source Edition (OSE),
+# as available from
+#
+#      http://www.openvstorage.org and
+#      http://www.openvstorage.com.
+#
+# This file is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Affero General Public License v3 (GNU AGPLv3)
+# as published by the Free Software Foundation, in version 3 as it comes
+# in the LICENSE.txt file of the Open vStorage OSE distribution.
+#
+# Open vStorage is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY of any kind.
+
+"""
+Some helpers
+"""
+
+from rest_framework.response import Response
+
+
+class OVSResponse(Response):
+
+    def __init__(self, data=None, status=200,
```

Comment:
Using a generic approach here, seems better to me
`super(OVSResponse, self).__init__(**kwargs)`